explainers/security-privacy-questionnaire-post-task.md

```diff
@@ -0,0 +1,154 @@
+# `scheduler.postTask()`: Security and Privacy Questionnaire Answers
+
+The following are the answers to the W3C TAG's
+[security and privacy self-review questionnaire](https://w3ctag.github.io/security-questionnaire/).
+
+> 01. What information might this feature expose to Web sites or other parties,
+>     and for what purposes is that exposure necessary?
+
+Tasks scheduled with this API have an associated priority, which is provided by
+consumers of the API. This priority could be obtained in JavaScript either by
+having access to the associated `TaskSignal`, or by monkey-patching the
+`scheduler` and intercepting calls to `postTask()`. The priority is used by the
+UA for scheduling decisions, and is fundamental to a prioritized scheduling
+API.
+
+Tasks also have an optional delay value &mdash; analogous to `setTimeout()`'s
+timeout value. This could similarly be obtained in JavaScript by
+monkey-patching the API. The delay value is required to schedule delayed tasks.
```

Comment:
The above two paragraphs aren't really about exposing information to the website, since it's the website itself providing this information. (In general, any values accessible by monkeypatching aren't really what this question is about.)

Reply:
I initially wanted to answer "No.", but the 3P tracking threat model section threw me off. These two paragraphs were setting that discussion up. That threat model section mentions pulling in and executing script, but didn’t specifically call out 3P contexts like other sections. "Third-party tracking" made me think of stuff like analytics libraries, so put together, I thought I needed to explore "what can be learned by 3P script from a 1P's usage of this API?" But I guess the relevant question for that threat model (for our case) is more "what can be learned by a 3P using the API?" --- which gets answered in Q9.